### PR TITLE
[Docker] update docker hub

### PR DESCRIPTION
**Pull request by QimingAi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +81,17 @@ bash scripts/download-data.sh
...
 ### 2. Installation
 
-:point_right: See [Run with Docker :whale:](docker/README.md) to build and install. 
+:point_right: Run with Docker :whale:
+
+The following command will pull the image [`hysia/hysia:v2o`](https://hub.docker.com/r/hysia/hysia/tags) automatically
+ if it doesn't exist on your local machine.
+
+```shell script
+docker run --gpus all -d -p 8000:8000 hysia/hysia:v2o
```
Comment (YuanmingLeee) [inline README.md:90]: add a `docker pull hysia/hysai:v2o` here

File: server/HysiaREST/settings.py [hunk 2]
```diff
@@ -25,11 +25,11 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 # We can access the remote
-ALLOWED_HOSTS = ['155.69.146.227', '155.69.146.227:3000']
+ALLOWED_HOSTS = ['155.69.145.185', '155.69.145.185:8000', '127.0.0.1', '127.0.0.1:8000']
```
Comment (YuanmingLeee) [inline server/HysiaREST/settings.py:28]: change to `ALLOWED_HOSTS = ['*']`

File: server/HysiaREST/settings.py [hunk 3]
```diff
@@ -25,11 +25,11 @@
...
 
 # CORS configs
 CORS_ORIGIN_ALLOW_ALL = False
-CORS_ORIGIN_WHITELIST = ['http://155.69.146.227:3000']
+CORS_ORIGIN_WHITELIST = ['http://155.69.145.185:8000']
```
Comment (YuanmingLeee) [inline server/HysiaREST/settings.py:32]: similar here

File: server/model_server/visual_model_server.py [hunk 4]
```diff
@@ -45,7 +45,7 @@
 
 # Path to frozen detection graph. This is the actual model that is used for the object detection.
 SSD_mobile_path = SERVER_ROOT + \
-                  '../../weights/ssd_mobilenet_v1_coco_2018_01_28/frozen_inference_graph.pb'
+                  '../../weights/ssd_mobilenet_v1_coco_2017_11_17/frozen_inference_graph.pb'
```
Comment (YuanmingLeee) [inline server/model_server/visual_model_server.py:48]: It supposed to be 2018 version, Is the pre-trained weight tar file not updated?

**Review by HuaizhengZhang — APPROVED** [review]

File: server/model_server/visual_model_server.py (hunk 4 above)
Comment (QimingAi) [reply server/model_server/visual_model_server.py:48]: yes, it is the 2017 version

Comment (YuanmingLeee) [reply server/model_server/visual_model_server.py:48]: I will update this

**Pull request merged**